### PR TITLE
Api list category document.

### PR DESCRIPTION
**Pull request by mvn-minhphan-dn**
## Description
Api list categories and its document.
## Reference Requirements
- Product Backlog ID: PBI-34
- Task ID: TI-1&2

## Reference Isuees
None

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
- routes/api.php

## SQL query and `explain`
-select `id`, `name`, (select count(*) from `books` where `categories`.`id` = `books`.`category_id` and `books`.`deleted_at` is null) as `books_count` from `categories` group by `id` limit 10 offset 0
image

### CODE REVIEW

File: docs/api-references/api_category.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+## Category Api
+
+### `GET` Categories
```
Comment (at-phutran2) [inline docs/api-references/api_category.md]: get là phương thức, còn hành động là ntn Minh ?? => `GET` List Categories

File: docs/api-references/api_category.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+```
+/api/categories
+```
+Get list all categories with paginate
+#### Request Headers
+| Key | Value |
+|---|---|
+|Accept|application\json
+
```
Comment (at-phutran2) [inline docs/api-references/api_category.md]: thêm request body
| Key | Value |
|---|---|
| content | Content |
| list_categories | text |

File: docs/api-references/api_category.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#### Response
+```json
```
Comment (at-phutran2) [inline docs/api-references/api_category.md]: thêm status : "meta": {
    "success": true,
    "code": 201
  }

**Review by at-phutran2 — COMMENTED** [review]
phải tách nhánh, docs và code riêng

File: routes/api.php [hunk 4]
```diff
@@ -12,3 +12,4 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
+Route::get('categories', 'Api\\CategoryController@index');
```
Comment (at-phutran2) [inline routes/api.php]: thiếu 1 hàng enter

**Review by at-phutran2 — COMMENTED** [review]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**